.github/workflows/common.yml

```diff
@@ -0,0 +1,75 @@
+---
+# See https://github.com/pharmaverse/admiraltemplate/tree/main/.github/workflows#readme
+name: Admiral Workflows
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - devel
+      - pre-release
+      - test
+  pull_request:
+    branches:
+      - main
+      - devel
+      - pre-release
+      - test
+  pull_request_review:
+    types: [submitted]
+  release:
+    types: [published]
+
+concurrency:
+  group: admiral-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
```

Comment:
I can not find the man-pages workflow. Did I miss it or have you removed it?

Reply:
It needs to be added to admiraltemplate, then it can be imported here. Separate task.

Reply:
I see

Reply:
I'll add that soon. Will keep this PR open until that one is ready.

Reply:
@bundfussr I have added the man pages workflow here and to the template.